README.md

```
@@ -1,22 +1,19 @@
 # Sutra
 
-** Offchain transaction builder framework for cardano using Elixr.**
+** Offchain transaction builder framework for cardano using Elixir.**
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)